bench/Benchmarks.cs

```diff
@@ -89,7 +83,7 @@ public static void Method() { }
     protected void Setup()
     {
         NodeEmbeddingPlatform platform = new(
-            LibnodePath,
+            null,
```

Comment:
Since it becomes optional, it is better to remove this parameter and instead make it a property for the `NodeEmbeddingPlatformSettings`